pyproject.toml

```diff
@@ -14,17 +14,14 @@ pydis_core = { version = "9.9.2", extras = ["async-rediscache"] }
 aiohttp = "3.8.4"
 arrow = "1.2.3"
 beautifulsoup4 = "4.12.2"
-colorama = { version = "0.4.6", markers = "sys_platform == 'win32'" }
```

Comment:
This is needed by `coloredlogs` for windows coloured logging. https://coloredlogs.readthedocs.io/en/latest/readme.html?highlight=colorama#optional-dependencies

Reply:
Yea I think we've got it this way around as we only want it installed on windows hosts, hence the markers, rather than marking it as an extra on colored logs so it installs for everyone

Reply:
Understood.
I've restored them both.